web/styles/blueslip.css

```diff
@@ -0,0 +1,153 @@
+.blueslip-display {
+    display: none;
+
+    &.show {
+        display: block;
+    }
+}
+
+.blueslip-animations {
+    &.show {
+        animation-name: fadeIn;
+        animation-duration: 0.3s;
+        animation-fill-mode: forwards;
+    }
+
+    &.fade-out {
+        animation-name: fadeOut;
+        animation-duration: 0.3s;
+        animation-fill-mode: forwards;
+    }
+}
+
+.blueslip-error-container {
+    /* We define this variable here since this is shared
+       with portico. */
+    --blueslip-overlay-translate-y-distance: 100px;
+    position: fixed;
+    /* Offset to account for the top padding + 5px from the top. */
+    top: calc(5px + (-1 * var(--blueslip-overlay-translate-y-distance)));
+    left: 50%;
+    transform: translateX(-50%);
+    width: 100%;
+    max-width: 900px;
+    z-index: 230;
```

Comment:
The `z-index` of the the alert popups is `220`, I have set the `z-index` of blueslip error overlay to `230` since we would probably want the blueslip errors to appear as the topmost element given its use-case.